### PR TITLE
Map a bunch of stuff in net.minecraft.server

### PR DESCRIPTION
**Pull request by NoahvdAa**

### CODE REVIEW

File: mappings/net/minecraft/block/entity/SkullBlockEntity.mapping [hunk 1]
```diff
@@ -16,11 +16,13 @@ CLASS net/minecraft/unmapped/C_vrhaavsg net/minecraft/block/entity/SkullBlockEnt
 	METHOD m_btvvujwr (Ljava/util/function/Consumer;Lcom/mojang/authlib/GameProfile;)V
 		ARG 1 profile
 	METHOD m_fanntxcn getOwner ()Lcom/mojang/authlib/GameProfile;
+	METHOD m_flpykord setup (Lnet/minecraft/unmapped/C_jyqwslie;Lcom/mojang/authlib/minecraft/MinecraftSessionService;Ljava/util/concurrent/Executor;)V
 	METHOD m_ikhwlxtp tick (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_vrhaavsg;)V
 		ARG 0 world
 		ARG 1 pos
 		ARG 2 state
 		ARG 3 blockEntity
+	METHOD m_klbqrovy clear ()V
```
Comment (IotaBread) [inline mappings/net/minecraft/block/entity/SkullBlockEntity.mapping]: Already mapped in #107

File: mappings/net/minecraft/server/rcon/QueryResponseHandler.mapping [hunk 2]
```diff
@@ -6,6 +6,7 @@ CLASS net/minecraft/unmapped/C_hfrdwddv net/minecraft/server/rcon/QueryResponseH
 	FIELD f_fawueshy port I
 	FIELD f_fcznabai GAME_ID Ljava/lang/String;
 	FIELD f_fnebczmm packetBuffer [B
+	FIELD f_hkftcmtf logger Lorg/slf4j/Logger;
```
Comment (triphora) [inline mappings/net/minecraft/server/rcon/QueryResponseHandler.mapping]: Constant, needs to be caps
```suggestion
	FIELD f_hkftcmtf LOGGER Lorg/slf4j/Logger;
```

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 3]
```diff
@@ -30,6 +37,8 @@ CLASS net/minecraft/unmapped/C_tljyhzqp net/minecraft/server/WorldStem
 			ARG 2 commandRegistrationEnvironment
 			ARG 3 level
 			ARG 4 safeMode
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 object
```
Comment (triphora) [inline mappings/net/minecraft/server/WorldStem.mapping]:
```suggestion
			ARG 1 o
```

File: mappings/net/minecraft/server/ServerConfigList.mapping [hunk 4]
```diff
@@ -8,6 +8,7 @@ CLASS net/minecraft/unmapped/C_ontrhiil net/minecraft/server/ServerConfigList
 	METHOD m_ceolyphy remove (Lnet/minecraft/unmapped/C_tvaqzxyk;)V
 		ARG 1 entry
 	METHOD m_coxkvypn contains (Ljava/lang/Object;)Z
+		ARG 1 object
```
Comment (IotaBread) [inline mappings/net/minecraft/server/ServerConfigList.mapping]:
```suggestion
		ARG 1 entry
```

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 5]
```diff
@@ -401,6 +421,17 @@ CLASS net/minecraft/server/MinecraftServer net/minecraft/server/MinecraftServer
 	METHOD m_zsmhozky getCurrentPlayerCount ()I
 	METHOD m_zsxirona getServerIp ()Ljava/lang/String;
 	METHOD m_zvxfodpo runOneTask ()Z
+	CLASS C_eaujwdwn RseourceManagerHolder
```
Comment (triphora) [inline mappings/net/minecraft/server/MinecraftServer.mapping]:
```suggestion
	CLASS C_eaujwdwn ResourceManagerHolder
```

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 6]
```diff
@@ -401,6 +421,17 @@ CLASS net/minecraft/server/MinecraftServer net/minecraft/server/MinecraftServer
...
+		FIELD f_nnwzxwwj resources Lnet/minecraft/unmapped/C_migzkpst;
+		FIELD f_scqzsofq resourceManager Lnet/minecraft/unmapped/C_carvbxzj;
+		METHOD <init> (Lnet/minecraft/unmapped/C_carvbxzj;Lnet/minecraft/unmapped/C_migzkpst;)V
+			ARG 1 resourceManager
+			ARG 2 resources
+		METHOD close close ()V
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 object
```
Comment (triphora) [inline mappings/net/minecraft/server/MinecraftServer.mapping]:
```suggestion
			ARG 1 o
```

File: mappings/net/minecraft/server/ServerMetadata.mapping [hunk 7]
```diff
@@ -18,6 +18,15 @@ CLASS net/minecraft/unmapped/C_bwttende net/minecraft/server/ServerMetadata
 		ARG 1 version
 	METHOD m_wkxexwvm setDescription (Lnet/minecraft/unmapped/C_rdaqiwdt;)V
 		ARG 1 description
+	CLASS C_cslkkyll ServerMetadataSerializer
```
Comment (IotaBread) [inline mappings/net/minecraft/server/ServerMetadata.mapping]: A bit redundant
```suggestion
	CLASS C_cslkkyll Serializer
```

File: mappings/net/minecraft/server/ServerMetadata.mapping [hunk 8]
```diff
@@ -18,6 +18,15 @@ CLASS net/minecraft/unmapped/C_bwttende net/minecraft/server/ServerMetadata
...
+		METHOD deserialize deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Ljava/lang/Object;
+			ARG 1 element
+			ARG 2 type
+			ARG 3 deserializationContext
+		METHOD serialize serialize (Ljava/lang/Object;Ljava/lang/reflect/Type;Lcom/google/gson/JsonSerializationContext;)Lcom/google/gson/JsonElement;
+			ARG 1 metadata
+			ARG 2 type
+			ARG 3 serializationContext
```
Comment (IotaBread) [inline mappings/net/minecraft/server/ServerMetadata.mapping]:
```suggestion
			ARG 3 context
```

File: mappings/net/minecraft/server/ServerMetadata.mapping [hunk 9]
```diff
@@ -18,6 +18,15 @@ CLASS net/minecraft/unmapped/C_bwttende net/minecraft/server/ServerMetadata
 		ARG 1 version
 	METHOD m_wkxexwvm setDescription (Lnet/minecraft/unmapped/C_rdaqiwdt;)V
 		ARG 1 description
+	CLASS C_cslkkyll ServerMetadataSerializer
+		METHOD deserialize deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Ljava/lang/Object;
+			ARG 1 element
+			ARG 2 type
+			ARG 3 deserializationContext
```
Comment (IotaBread) [inline mappings/net/minecraft/server/ServerMetadata.mapping]:
```suggestion
			ARG 1 json
			ARG 2 type
			ARG 3 context
```

File: mappings/net/minecraft/server/ServerMetadata.mapping [hunk 10]
```diff
@@ -47,9 +57,11 @@ CLASS net/minecraft/unmapped/C_bwttende net/minecraft/server/ServerMetadata
 		METHOD m_gxeobiyp getSample ()[Lcom/mojang/authlib/GameProfile;
 		METHOD m_jvjhzktb getOnlinePlayerCount ()I
 		CLASS C_dmwbfyrz Deserializer
-			METHOD deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Ljava/lang/Object;
+			METHOD deserialize deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Ljava/lang/Object;
 				ARG 1 json
-			METHOD serialize (Ljava/lang/Object;Ljava/lang/reflect/Type;Lcom/google/gson/JsonSerializationContext;)Lcom/google/gson/JsonElement;
+				ARG 2 type
+				ARG 3 deserializationContext
```
Comment (IotaBread) [inline mappings/net/minecraft/server/ServerMetadata.mapping]:
```suggestion
				ARG 3 context
```

File: mappings/net/minecraft/server/MinecraftServer.mapping (hunk 5 above)
Comment (NoahvdAa) [reply mappings/net/minecraft/server/MinecraftServer.mapping]: Whoops 😅

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 11]
```diff
@@ -171,6 +184,9 @@ CLASS net/minecraft/server/MinecraftServer net/minecraft/server/MinecraftServer
 		ARG 1 worldSavePath
 	METHOD m_hrqzcsgt dumpThreads (Ljava/nio/file/Path;)V
 		ARG 1 path
+	METHOD m_hrskcghu (Ljava/nio/file/Path;)V
+		ARG 0 ignored
+	METHOD m_hsdjwegl hidesOnlinePlayers ()Z
```
Comment (IotaBread) [inline mappings/net/minecraft/server/MinecraftServer.mapping]:
```suggestion
	METHOD m_hsdjwegl shouldHideOnlinePlayers ()Z
```
?

File: mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping [hunk 12]
```diff
@@ -53,3 +56,22 @@ CLASS net/minecraft/unmapped/C_kefepfwh net/minecraft/server/dedicated/ServerPro
 	METHOD m_rkjqubss (Ljava/lang/Integer;)Ljava/lang/Integer;
 		ARG 0 percentage
 	METHOD m_vcawwowm load (Ljava/nio/file/Path;)Lnet/minecraft/unmapped/C_kefepfwh;
+		ARG 0 path
+	METHOD m_xxhdqysd getGeneratorOptions (Lnet/minecraft/unmapped/C_wqxmvzdq;)Lnet/minecraft/unmapped/C_oxnufhsj;
+		ARG 1 registryManager
+	CLASS C_swovxaqc WorldGeneratorProperties
+		FIELD f_hsgwowdc seed Ljava/lang/String;
+		FIELD f_kbscfzuf generationSettings Lcom/google/gson/JsonObject;
+		FIELD f_lupnzgbk levelType Ljava/lang/String;
```
Comment (IotaBread) [inline mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping]: This should probably be worldType

File: mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping [hunk 13]
```diff
@@ -53,3 +56,22 @@ CLASS net/minecraft/unmapped/C_kefepfwh net/minecraft/server/dedicated/ServerPro
...
+		FIELD f_zdbkpgmx generateStructures Z
+		METHOD <init> (Ljava/lang/String;Lcom/google/gson/JsonObject;ZLjava/lang/String;)V
+			ARG 1 seed
+			ARG 2 generationSettings
+			ARG 3 generateStructures
+			ARG 4 levelType
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 object
```
Comment (IotaBread) [inline mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping]:
```suggestion
			ARG 1 o
```

File: mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping [hunk 14]
```diff
@@ -53,3 +56,22 @@ CLASS net/minecraft/unmapped/C_kefepfwh net/minecraft/server/dedicated/ServerPro
...
+		METHOD m_dgpgrrdu levelType ()Ljava/lang/String;
```
Comment (IotaBread) [inline mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping]: See other comment

File: mappings/net/minecraft/server/world/BlockEvent.mapping [hunk 15]
```diff
@@ -10,3 +10,7 @@ CLASS net/minecraft/unmapped/C_dnmqbjtb net/minecraft/server/world/BlockEvent
 		ARG 4 data
 	METHOD equals (Ljava/lang/Object;)Z
 		ARG 1 o
+	METHOD m_cifaeodo getData ()I
+	METHOD m_iqwpwuux getBlock ()Lnet/minecraft/unmapped/C_mmxmpdoq;
+	METHOD m_sbwkdmfx getPos ()Lnet/minecraft/unmapped/C_hynzadkk;
+	METHOD m_wawjronn getType ()I
```
Comment (IotaBread) [inline mappings/net/minecraft/server/world/BlockEvent.mapping]: If the class is a record these shouldn't have `get` prefix

**Review by OroArmor — COMMENTED** [review]
Get the other requests and it's good

**Review by IotaBread — APPROVED** [review]

**Review by triphora — APPROVED** [review]

**Review by NoComment1105 — APPROVED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

**Pull request merged**